Makefile

```diff
@@ -66,6 +67,12 @@ clean-docs:
 .PHONY : test-python test-builtin test-suite
 test-python: test-builtin test-suite
 
+test-python-mpich-cray: 
+	srun -n 2 $(PYTHON) $(MPITESTRUNNER)
+
+test-python-mpi: 
+	mpirun -np 2 $(PYTHON) $(MPITESTRUNNER)
+
```

Comment:
This is not really a `mpich-cray` thing, I think `srun -n 2` vs `mpirun -np 2` should be dealt with `mpi_launcher` and `num_proc` Makefile variables.

Reply:
Sure!